### ISSUE TITLE
 hammr scan build fails

### ISSUE THREAD

**Issue by Yorche** [usharesoft/hammr#143]
Even if a scan ID which exists in the "hammr scan list" is specified, "hammr scan build" raises "ERROR: scan not found".
- Environments
  AppCenter3.6-4
  hammr-3.6==1.0
  uforge-python-sdk==3.6.0.9

```
[root@Hammr36 hammr_json]# hammr scan list
INFO: no username nor password provided on command line, trying credentials file
INFO: Using credentials file: /root/.hammr/credentials.json
INFO: Using url http://10.124.111.124/api
Getting scans for [tanaka] ...
+----+---------------------------------------------+--------+-----------------+
| Id |                    Name                     | Status |  Distribution   |
+====+=============================================+========+=================+
| 13 | Test-CentOS-6-Scan-by-Hammr                 |        | CentOS 6 x86_64 |
+----+---------------------------------------------+--------+-----------------+
| 23 |         Test-CentOS-6-Scan-by-Hammr Scan #1 | Done   |                 |
+----+---------------------------------------------+--------+-----------------+
| 11 | ScanTestCentOS6.7                           |        | CentOS 6 x86_64 |
+----+---------------------------------------------+--------+-----------------+
| 17 |         ScanTestCentOS6.7 Scan #1           | Done   |                 |
+----+---------------------------------------------+--------+-----------------+
| 4  | test-cent6                                  |        | CentOS 6 x86_64 |
+----+---------------------------------------------+--------+-----------------+
| 4  |         test-cent6 Scan #1                  | Done   |                 |
+----+---------------------------------------------+--------+-----------------+


[root@Hammr36 hammr_json]# hammr scan build --id 23 --file scan-image-ovf.json
INFO: no username nor password provided on command line, trying credentials file
INFO: Using credentials file: /root/.hammr/credentials.json
INFO: Using url http://10.124.111.124/api
Validating the template file [scan-image-ovf.json] ...
OK: Syntax of template file [scan-image-ovf.json] is ok
ERROR: scan not found
```

My  json file providing the builder parameters is: 

```
{
  "builders": [
    {
      "type": "OVF or OVA",
      "hardwareSettings": {
        "memory": 1024,
        "hwType": 7
      },
      "installation" : {
        "diskSize" : 1000
      }
    }
  ]
}
```

**Comment by lqueiroga** [comment]
I think this was already fixed but not merge see Pull Request 123 = FIX #122 "scan build"error : scan not found

**Comment by jgweir** [comment]
Fixed by #123 -- reopen if not the case.
